### PR TITLE
Add types for fs-plus 3.0

### PR DESCRIPTION
**Pull request by unindented**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/fs-plus/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,305 @@
+// Type definitions for fs-plus 3.0
+// Project: http://atom.github.io/fs-plus
+// Definitions by: My Self <https://github.com/me>
```
Comment (plantain-00) [inline types/fs-plus/index.d.ts]: It should be your name and github url here.

File: types/fs-plus/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,305 @@
+// Type definitions for fs-plus 3.0
+// Project: http://atom.github.io/fs-plus
```
Comment (plantain-00) [inline types/fs-plus/index.d.ts]: Usually use https://github.com/atom/fs-plus as project url

**Review by plantain-00 — CHANGES_REQUESTED** [review]

File: types/fs-plus/index.d.ts (hunk 1 above)
Comment (unindented) [reply types/fs-plus/index.d.ts]: Oops sorry, I created a bunch of type definitions today, and skipped this. I'll fix.

**Comment by unindented** [comment]
@plantain-00 @typescript-bot all done!

**Review by plantain-00 — APPROVED** [review]

**Pull request merged**